html5/render/vue/modules/dom.js

```diff
@@ -95,60 +82,49 @@ const dom = {
    *   ps: scroll-to has 'ease' and 'duration'(ms) as options.
    */
   scrollToElement: function (vnode, options) {
-    const { isArray } = utils
+    const {
+      isArray
+    } = utils
     if (isArray(vnode)) {
       if (process.env.NODE_ENV === 'development') {
         console.warn('[vue-render] the ref passed to animation.transitionOnce is a array.')
       }
       vnode = vnode[0]
     }
-
     const scroller = getParentScroller(vnode)
     const scrollDirection = scroller && scroller.scrollDirection || 'vertical'
-
     const isWindow = !scroller
     const ct = isWindow ? document.body : scroller.$el
     const el = vnode.$el || vnode.elm
-
     if (ct && el) {
       // if it's a list, then the listVnode.scrollDirection is undefined. just
       // assum it is the default value 'vertical'.
       const dSuffix = ({
         horizontal: 'Left',
         vertical: 'Top'
       })[scrollDirection]
-
       const ctRect = ct.getBoundingClientRect()
       let elRect = el.getBoundingClientRect()
-
-      /**
-       * if it's a waterfall, and you want to scroll to a header, then just
-       * scroll to the top.
-       */
-      if (scroller
-        && scroller.weexType === 'waterfall'
-        && scroller._headers
-        && scroller._headers.indexOf(vnode.$vnode || vnode) > -1) {
+        /**
+         * if it's a waterfall, and you want to scroll to a header, then just
+         * scroll to the top.
+         */
+      if (scroller && scroller.weexType === 'waterfall' && scroller._headers && scroller._headers.indexOf(vnode.$vnode || vnode) > -1) {
```

Comment:
This is way too long for one line code. It's hard to read if it exceeds 80 letters.